centreon/www/front_src/src/Main/index.tsx

```diff
@@ -52,11 +52,13 @@ const Main = (): JSX.Element => {
   );
   const setIsOnPublicPageAtom = useSetAtom(isOnPublicPageAtom);
 
-  const navigateTo = (path: string): void => {
+  const navigateTo = (path: string): string => {
     navigate(path);
     window.location.reload();
   };
 
+  console.log('hello darkness my old friend');
```

Comment:
🚫 **[Biome]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don't use console
```
  58 │   };
  59 │
> 60 │   console.log('hello darkness my old friend');
     │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  61 │
  62 │   startTransition(() => {
```
i Usage of console is disallowed.
i Unsafe fix: Remove console
```
   57  57 │       window.location.reload();
   58  58 │     };
   59     │ -
   60     │ - ··console.log('hello·darkness·my·old·friend');
   61  59 │
   62  60 │     startTransition(() => {
```
<!-- __reviewdog__:ChAzOTJmZWUyYjQ1MmM0ODViEgVCaW9tZQ== -->